spec/jara/releaser_spec.rb

```diff
@@ -156,6 +162,18 @@ module Jara
         command.should_not be_empty
       end
 
+      it 'allows branch to be specified in options' do
+        options[:branch] = 'another_branch'
+        staging_releaser.build_artifact
+        command = executed_commands.grep(/^git archive .+ #{another_branch_sha}/).first
+        command.should_not be_empty
+      end
+
+      it 'Do not allow branch to be specified for production environment' do
+        options[:branch] = 'another_branch'
+        expect { production_releaser.build_artifact }.to raise_error(JaraError, /master/i)
```

Comment:
Assert on more of the error message, for example `/must be released from master/i`.